packages/shared/components/atoms/tiles/TransactionActivityTile.svelte

```diff
@@ -137,67 +168,66 @@
                     <Text
                         fontWeight={FontWeight.semibold}
                         lineHeight="140"
-                        color={data.direction === ActivityDirection.In ? 'blue-700' : ''}
+                        color={isIncoming ? 'blue-700' : ''}
                         classes="whitespace-nowrap"
                     >
                         {amount}
                     </Text>
                 </div>
-
                 <div class="flex flex-row justify-between">
                     <Text fontWeight={FontWeight.medium} lineHeight="140" color="gray-600">
-                        {localize(
-                            data.direction === ActivityDirection.In ? 'general.fromAddress' : 'general.toAddress',
-                            { values: { account: subjectLocale } }
-                        )}
+                        {localize(isIncoming ? 'general.fromAddress' : 'general.toAddress', {
+                            values: { account: subjectLocale },
+                        })}
                     </Text>
                     <Text fontWeight={FontWeight.medium} lineHeight="140" color="gray-600" classes="whitespace-nowrap">
                         {fiatAmount}
                     </Text>
                 </div>
             </div>
-        </div>
-        {#if isTimelocked || (data.isAsync && (data.direction === ActivityDirection.Out || !data.isClaimed))}
+        </info-container>
+        {#if isAsyncActivity}
             <HR />
-            <div class="flex w-full justify-between space-x-4">
-                <div class="flex flex-row justify-center items-center space-x-2">
-                    {#if !data.isClaimed || isTimelocked}
-                        <Icon width="16" height="16" icon="timer" classes="text-gray-600" />
+            <async-activity-container class="flex w-full justify-between space-x-4">
+                <info-container class="flex flex-row justify-center items-center space-x-2">
+                    {#if isUnclaimed || isTimelocked}
+                        <TooltipIcon
+                            icon={isTimelocked ? IconEnum.Timelock : IconEnum.ExpirationTime}
+                            classes="text-gray-600 dark:text-gray-200"
+                            title={localize(`general.${isUnclaimed ? 'expirationTime' : 'timelockDate'}`)}
+                            text={asyncStatusTooltipText}
+                            position={Position.Top}
```

Comment:
the position is slighlty off. Maybe you need to include the anchor element?

Reply:
The anchor is not exported, if you check the TooltipIcon you will see that the anchor is the icon container

Reply:
anchor was just a maybe-solution. But is the tooltip also off on ur side?

Reply:
Oh yes, it is happening sorry! That is because the TooltipIcon component needed a container tag, so the outside "space-x-2" class wouldn't affect the tooltip margins

Reply:
Sharp eyes :eyeglasses:

Reply:
sneaky